docs/style_guide.md

```diff
@@ -0,0 +1,60 @@
+# Contributing to Bystro
+
+[WIP]
+
+Welcome to Bystro! We're glad you're interested in contributing to our project. Before you start,
+please take a moment to familiarize yourself with our approach and guidelines.
+
+## Installing Bystro
+[TK]
```

Comment:
I think the installation guide should be its own doc - maybe just put a link here? Same in contributing doc.

Reply:
added link to pre-existing installation guide in CONTRIBUTING.md